src/main/java/fi/okm/jod/yksilo/entity/Koulutus.java

```diff
@@ -39,6 +42,7 @@
 
 @Entity
 @Getter
+@Table(indexes = {@Index(columnList = "yksilo_id")})
 public non-sealed class Koulutus implements OsaamisenLahde {
```

Comment:
Generic comment: Should edudation have start and end date? End date as optional. There is a plan to have also integration to TMT. There datamodel has fields for the  end date and aborted date which are depending of the state of the education e.g. "in progress", "graduated", "aborted" etc.

Reply:
Yes, Koulutus already has alkuPvm and loppuPvm (both currently optional), but the state is not (yet) modeled. We could add that now, or in some later iteration (not sure if that is fully thought out yet).